### PR TITLE
Unsplash searchbar

### PR DESCRIPTION
**Pull request by hannah-rackley**

### CODE REVIEW

File: src/Frontend/Components/SearchBarFormContainer.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      this.state = {
+          newInput: "",
+      }
+  }
+  render() {
+    let searchImages = (event) => {
+      event.preventDefault();
+      let input = event.target.children[0].value;
+      this.setState({ newInput: "" });
+      fetch(`/search/${input}`, {
```
Comment (sarathrasher) [inline src/Frontend/Components/SearchBarFormContainer.js:17]: So when I pulled this down and tried to search, the onChange worked for the input box, but it didn't make a fetch request to the backend. Is that where it's supposed to be currently?

Comment (hannah-rackley) [reply src/Frontend/Components/SearchBarFormContainer.js:17]: I haven't hooked up the backend yet! So it makes sense that it isn't returning anything

**Comment by TRACYMUSIKER** [comment]
it gets a console log when searching! Thanks Hannah! Looks good!

**Review by TRACYMUSIKER — APPROVED** [review]
👍

**Pull request merged**